docs/technical_reference/unit_models/treatment_models/chem_soft.rst

```diff
@@ -0,0 +1,255 @@
+   "Carbon dioxide second stage", "Alkalinity + Soda ash dose - Source total hardness + Residual hardness"
+
+The following equations are independent of the softening procedure selected but depend on the feed composition.
+
+.. csv-table::
+   :header: "Description", "Variable Name", "Symbol", "Equation"
+
+   "MgCl2 dose (if silica removal is selected)", "``mgcl2_dosing``", ":math:`MgCl_{2}`", ":math:`Ratio_{MgCl_{2}/SiO_{2}} * SiO_{2}` "
+   "Sludge produced", "``sludge_prod``", ":math:`m_{sludge}`",  ":math:`Q_{feed} * (\text{Ca-SF}_{CaCO_{3}-hardness} * Ca_{CaCO_{3}-hardness} + \text{Mg-SF}_{CaCO_{3}-hardness} * Mg_{CaCO_{3}-hardness} + Ca_{non-CaCO_{3}-hardness} + \text{Mg-SF}_{non-CaCO_{3}-hardness} * Mg_{non-CaCO_{3}-hardness} + \text{Excess CaO} + TSS + MgCl_{2})`"
+   "Volume of mixer", "``volume_mixer``", ":math:`V_{mixer}`", ":math:`Q_{feed} * RT_{mixer} * n_{mixer}`"
+   "Volume of flocculator", "``volume_floc``", ":math:`V_{floc}`", ":math:`Q_{feed} * RT_{floc} * n_{floc}`"
+   "Volume of sedimentation basin", "``volume_sed``", ":math:`V_{sed}`", ":math:`Q_{feed} * RT_{sed}`"
+   "Volume of recarbonation basin", "``volume_recarb``", ":math:`V_{recarb}`", ":math:`Q_{feed} * RT_{recarb}`"
+
+Costing
+---------
+
+The following table lists out the coefficients used in the equations to calculate the capital and operating costs
+for the mixer, flocculator, sedimentation basin and recarbonation basin. The coefficients are assigned as mutable Parameters.
+
+.. csv-table::
+   :header: "Unit", "Variable Name", "``_constant``", "``_coeff/_coeff_1``", "``_coeff_2``", "``_coeff_3``", "``_exp/_exp_1``", "``_exp_2``"
+
+   "**Capital**", "", "", "", "", "", "", ""
+   "Mixer", "``mix_tank_capital``", "28584", "0.0002","22.776","", "2", "" 
+   "Flocculator", "``floc_tank_capital``", "217222", "673894", "", "", "", ""
+   "Sedimentation basin", "``sed_basin_capital``", "182801", "-0.0005", "86.89", "", "2", ""
+   "Recarbonation basin", "``recarb_basin_capital``", "19287", "4e-9", "-0.0002", "10.027", "3", "2"
+   "Recarbonation basin source", "``recarb_basin_source_capital``", "130812", "9e-8", "-0.001", "42.578", "", "2"
+   "Lime feed system", "``lime_feed_system_capital``", "193268", "20.065", "", "", "", ""
+   "Administrative capital", "``admin_capital``", "", "69195", "", "", "0.5523", ""
+   "**Operating**", "", "", "", "", "", "", ""
+   "Mixer", "``mix_tank_op``", "22588", "-3e-8","0.0008","2.8375", "3", "2" 
+   "Flocculator", "``floc_tank_op``", "6040", "3e-13", "-4e-7", "0.318", "3", "2"
+   "Sedimentation basin", "``sed_basin_op``", "6872", "7e-10", "-0.00005", "1.5908", "3", "2"
+   "Recarbonation basin", "``recarb_basin_op``", "10265", "1e-8", "-0.0004", "6.19", "3", "2"
+   "Lime feed system", "``lime_feed_system_op``", "", "4616.7", "", "", "0.4589", ""
+   "Lime sludge management system", "``sludge_disposal_cost``", "", "35", "", "", "", ""
+   "Administrative Operational", "``admin_op``", "", "88589", "", "", "0.4589", ""
```

Comment:
What's the source of these costs? I would include it (them).

Reply:
Add the reference in the initial text.